Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -58,6 +58,7 @@ namespace DriverDetails
 		{OS_WINDOWS,VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_LINUX,  VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_WINDOWS,VENDOR_INTEL,    DRIVER_INTEL,        -1, BUG_INTELBROKENBUFFERSTORAGE, 101810.3907, 101810.3960, true},
+		{OS_ALL,    VENDOR_MESA,     DRIVER_I965,         -1, BUG_BROKENIMAGELOADSTORE,-1.0, -1.0, true},
```

Comment:
How can I test and confirm bug presence on Sandybridge? I understand I'd have to set "early depth on"?